setup.py

```diff
@@ -97,7 +96,14 @@ def read(path, encoding="utf-8"):
 ### NEEDED FOR A BATTERIES-LOADED EXPERIENCE
 udf_libs = [
     "facenet-pytorch>=2.5.2",
-    "easyocr>=1.5.0"
+    "easyocr>=1.5.0",
+    "ipython",
```

Comment:
Are these requirements of `yolo`? Can we get remove some of these?

Reply:
I have removed a few dependencies